imagetool/src/test/java/com/oracle/weblogic/imagetool/integration/BaseTest.java

```diff
@@ -316,4 +317,14 @@ private static void checkCmdInLoop(String cmd, String matchStr)
             }
         }
     }
+
+    private static String getImagetoolZipfile() throws Exception {
+        final File targetDir = new File(getTargetDir());
+        for (final File f : targetDir.listFiles()) {
+            if (f.isFile() && f.getName().endsWith("SNAPSHOT.zip")) {
```

Comment:
This won't work when we do release versions that don't have the SNAPSHOT suffix.  Why don't you pass the version number from the maven pom file as a system property?

Reply:
pushed another commit. Please review.
Thanks!